packages/cascara/src/ui/Table/style.css

```diff
@@ -0,0 +1,22 @@
+table {
+  text-align: left;
+  position: relative;
+  border-collapse: collapse;
+}
+th,
+td {
+  padding: 0.25rem;
+}
+th {
+  background: white;
+  position: sticky;
```

Comment:
I saw in your example this is working btw.

When. I tried doing this a little over a year ago, when doing position on any part of the table, it removed it from the flow of the document and the column widths were no longer tied to the width of the columns below in the table. It I think was not a problem in all browsers. I do not recall which worked or did not. Might have only been wrong in Chrome. Seems good now though.